### PR TITLE
[Aaron Saw Min Sern] iP

### PR DESCRIPTION
**Pull request by aaronsms**

### CODE REVIEW

File: src/main/java/duke/Storage.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	}
+
+    public void createDirectoryAndFile() {
+        try {
+            final int i = filePath.lastIndexOf("/");
+            String dirPath = filePath.substring(0, i);
+
+            Path dir = Paths.get(dirPath);
+            Path path = Paths.get(filePath);
+            if (!Files.exists(dir))
```
Comment (ssagit) [inline src/main/java/duke/Storage.java]: [Code Standard] Should have {} for all if() statements

File: src/main/java/duke/Storage.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            Files.createFile(path);
+        } catch (IOException e) {
+            System.err.println("Parent directory does not exist.");
+            e.printStackTrace();
+        }
+    }
+
+    public void saveFile(List<String> data) {
+        Path path = Paths.get(filePath);
+        if (!Files.exists(path))
```
Comment (ssagit) [inline src/main/java/duke/Storage.java]: [Code Standard] Should have {} for all if() statements

**Review by ssagit — COMMENTED** [review]
Very minor coding standard violations (only in Duke.java), missing javadocs (perhaps not done yet with A-JavaDoc yet?

**Comment by aaronsms** [comment]
@ssagit Thanks for the review. Indeed, please give me some time to be come up with A-JavaDoc, will bump you when it is PRed.

**Comment by aaronsms** [comment]
@ssagit Yeah it is up!

**Pull request closed without merging**